.github/workflows/helm-lint.yml

```diff
@@ -20,14 +20,17 @@
 name:  SDE Lint and Test Charts
 
 on:
+  ### Trigger the workflow on push on main branch and path charts/**
```

Comment:
do you really need the comments? was just a litle idea for you to understand... 😉

Reply:
i am ok with it, will remove in next pr.

Reply:
removed comments.